web/src/main/webapp/WEB-INF/web.xml

```diff
@@ -1,410 +1,272 @@
+		<param-value>true</param-value>
 	</context-param>
-
-<!-- <context-param>
-<param-name>com.sun.faces.responseBufferSize</param-name>
-<param-value>1048576</param-value>
-</context-param>     --> 
-      <context-param>
+	<context-param>
+		<param-name>com.sun.faces.numberOfViewsInSession</param-name>
+		<param-value>50</param-value>
+	</context-param>
+	<context-param>
+		<param-name>com.sun.faces.numberOfLogicalViews</param-name>
+		<param-value>50</param-value>
+	</context-param>
+	<context-param>
+		<param-name>com.sun.faces.serializeServerState</param-name>
+		<param-value>false</param-value>
+	</context-param>
+	<context-param>
 		<param-name>javax.faces.FACELETS_BUFFER_SIZE</param-name>
 		<param-value>1048576</param-value>
-	  </context-param>
-      
-<context-param>
-<param-name>com.sun.faces.defaultResourceMaxAge</param-name>
-<param-value>86400</param-value>
-</context-param>      
-      
+	</context-param>
+	<context-param>
+		<param-name>com.sun.faces.defaultResourceMaxAge</param-name>
+		<param-value>86400</param-value>
+	</context-param>
 	<context-param>
-        <param-name>javax.faces.STATE_SAVING_METHOD</param-name>
-        <param-value>server</param-value>
+		<param-name>javax.faces.STATE_SAVING_METHOD</param-name>
+		<param-value>server</param-value>
 	</context-param>
-    
-    <context-param>
-    	<param-name>javax.faces.PARTIAL_STATE_SAVING</param-name>
-    	<param-value>true</param-value>
-    </context-param>
-    
-    
-    <listener>
+	<context-param>
+		<param-name>javax.faces.PARTIAL_STATE_SAVING</param-name>
+		<param-value>true</param-value>
+	</context-param>
+	<listener>
 		<listener-class>org.phoenixctms.ctsms.web.LogoutListener</listener-class>
 	</listener>
-	
 	<context-param>
-       <param-name>javax.faces.FACELETS_LIBRARIES</param-name>
-       <param-value>/WEB-INF/ctsms-taglib.xml</param-value>
-    </context-param>
-    
-    
-    
-  <servlet>
-    <servlet-name>Jersey REST Service</servlet-name>
-    <servlet-class>com.sun.jersey.spi.container.servlet.ServletContainer</servlet-class>
-             <init-param>
-            <param-name>javax.ws.rs.Application</param-name>
-            <param-value>org.phoenixctms.ctsms.web.jersey.Application</param-value>
-        </init-param> 
-    <init-param>
-        <param-name>com.sun.jersey.spi.container.ContainerResponseFilters</param-name>
-        <param-value>org.phoenixctms.ctsms.web.jersey.provider.CORSFilter</param-value>
-    </init-param>    
-	<init-param>
-	  <param-name>com.sun.jersey.spi.container.ContainerRequestFilters</param-name>
-	  <param-value>org.phoenixctms.ctsms.web.jersey.provider.TrustedHostFilter</param-value>
-	</init-param>    
-    <load-on-startup>20</load-on-startup>
-  </servlet>
-  <servlet-mapping>
-    <servlet-name>Jersey REST Service</servlet-name>
-    <url-pattern>/rest/*</url-pattern> <!-- see WebUtil.REST_API_PATH -->
-  </servlet-mapping>
-  
-  
-    <servlet>
```

Comment:
is there a option to explicitly turn off swagger if it is auto-loaded nowadays?

Reply:
This alteration doesn't change the way swagger is currently loaded - it was autoloaded also before this change